stylesheets/commons/Brackets.less

```diff
@@ -1153,12 +1153,6 @@ img.brkts-champion-icon,
 	font-size: smaller;
 	line-height: 1;
 	margin-left: 4px;
-}
-
-.block-player > .note,
-.block-team > .note,
-.block-literal > .note {
-	align-self: flex-start;
 	font-weight: bold;
```

Comment:
```suggestion
```

Reply:
imho, I think it looks better with the bold, the note after the table is also bolded

Reply:
Then we should match the font size instead, no?